documentation/config_logging_advanced.md

```diff
@@ -4,123 +4,133 @@ title: "Puppet Server: Advanced Logging Configuration"
 canonical: "/puppetserver/latest/config_logging_advanced.html"
 ---
 
-Puppet Server uses the very flexible, very powerful [Logback](http://logback.qos.ch/) library under the hood to handle all of our logging needs.  Logback can be configured via the `logback.xml` file, which is located at `/etc/puppetlabs/puppetserver/logback.xml` by default.
+Puppet Server uses the [Logback](http://logback.qos.ch/) library to handle all of its logging. Logback configuration settings are stored in the [`logback.xml`](./config_file_logbackxml.html) file, which is located at `/etc/puppetlabs/puppetserver/logback.xml` by default.
```

Comment:
Either extension works on the docs site since we redirect URLs ending in .md to .html. We generally use .html for links within the docs site (including other docs-site links in this doc).